exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -335,6 +351,50 @@ private void javaPropertiesToSystemOptions() {
     }
   }
 
+
+  // Polls for graceful file to check if graceful shutdown is triggered from the script.
+  private static class PollShutdownThread extends Thread {
```

Comment:
Rename to `GracefulShutdownThread`

Reply:
Changed it. Initially, it was GracefulShutdownThread but I thought people might be confused since it only gracefully shuts down from the script.